### ISSUE TITLE
better error message when fastq quality value above qmax 41

### ISSUE THREAD

**Issue by frederic-mahe** [torognes/vsearch#368]
building on the issue reported by Laurie Agosto (https://groups.google.com/forum/?#!topic/vsearch-forum/03lC9IpMPY0), I think vsearch could output a more helpful error message when Q-values are higher than 41:
```sh
## J represents a Q-value of 41
printf "@s\nA\n+\nJ\n" | vsearch --fastq_filter - --fastqout /dev/null --quiet
```
all good.
```sh
## K represents a Q-value of 42
printf "@s\nA\n+\nK\n" | vsearch --fastq_filter - --fastqout /dev/null --quiet
```
```
Fatal error: FASTQ quality value (42) above qmax (41)
```
I suggest to add a few lines to the error message, something like this:
```
Normal Q-values range from 0 to 41.
To allow higher Q-values please use --fastq_qmax 42
```
Please feel free to improve.

**Comment by torognes** [comment]
Added in commit 6ccd8204aae89f47d2db4b8855c05ec85b439b31 for commands fastx_filter, fastq_eestats and fastq_mergepairs.

**Comment by torognes** [comment]
Fixed in vsearch version 2.13.0, just released.
